src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,75 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { useAppDispatch } from '../../app/hooks';
+import { actions } from '../../features/currentTodo';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  todo: Todo;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    getUser(todo?.userId).then(data => {
```

Comment:
You are not handling the potential error from the 'getUser' promise. This can lead to unhandled promise rejections. You should add a '.catch()' block to handle potential errors.